package.json

```diff
@@ -36,11 +40,15 @@
     "moment-timezone": "^0.5.33",
     "query-string": "^7.0.1",
     "react": "^16.13.1",
+    "react-autosuggest": "^10.1.0",
     "react-dom": "^16.13.1",
     "react-helmet-async": "^1.1.2",
+    "react-instantsearch-dom": "^6.25.0",
+    "react-instantsearch-hooks-web": "^6.25.0",
     "react-moment": "^0.9.6",
     "refractor": "^2.10.0",
     "rxjs": "^6.5.2",
-    "slugify": "^1.6.0"
+    "slugify": "^1.6.0",
+    "styled-components": "^5.3.5"
```

Comment:
I think one of the main pieces of work going forward on this will be to eliminated any styled-component dependencies, since we use a different styling solution (material-ui v4 and step-ui). That may be worth starting to dig into... how you will rewrite the included components based on styled-components using material-ui and step-ui. Will work offline to help you get set up on step-ui! Here are the docs for material-ui. Be sure to reference v4 docs, not v5 (we're not upgraded yet):

https://v4.mui.com/